wiki/Music_theory/Time_signature/en.md

```diff
@@ -0,0 +1,17 @@
+---
+stub: true
+tags:
+  - music theory
+  - mapping
+  - beatmapping
+  - tips
+  - sheet music
+---
+
+# Time signature
+
+*For more information on time signatures in music theory, see: [Time signature](https://en.wikipedia.org/wiki/Time_signature)*
+
+A **time signature** is a musical term that indicates the number of [beats](/wiki/Beat) in a single measure (indicated by the top number), and which note indicates a single beat (indicated by the bottom number).
+
+osu! offers two alternative time signatures for [mapping](/wiki/Beatmapping), which can be selected from the `Timing` menu in the [beatmap editor](/wiki/Client/Beatmap_editor). These include 4/4 (a.k.a. "common time" signature), and 3/4 (a.k.a. the "Waltz" or "three-four-time" signature). osu! will not prevent users from mapping songs that are written in different signatures, although it is not guaranteed that the beatmap will align with the song if this is done.
```

Comment:
can't really understand what the last sentence means:
- osu! also provides 5/4, 6/4, and 7/4 time signatures from the editor
- "it is not guaranteed that the beatmap will align with the song" -- it's a mapper's task to make sure a beatmap follows the song, did you maybe want to say something like "songs with other signatures are disappearingly rare"?

Reply:
i didn't write the article, just moved it